x/datadeal/keeper/deal.go

```diff
@@ -213,6 +213,22 @@ func (k Keeper) SellData(ctx sdk.Context, msg *types.MsgSellData) error {
 	return nil
 }
 
+func (k Keeper) checkDataSaleStatus(ctx sdk.Context, sellerAddress, dataHash string, dealID uint64) error {
+	getDataSale, _ := k.GetDataSale(ctx, dataHash, dealID)
```

Comment:
Because if the first data requested to sell, there will be return an error.

Reply:
Then, I think it would be good to handle errors explicitly.

```suggestion
	getDataSale, err := k.GetDataSale(ctx, dataHash, dealID)
	if err != nil {
	    if errors.Is(types.ErrDataSaleNotFound, err) {
	    	    return nil
	    } else {
		    return err
	    }
	}
	
```

Reply:
I agree :)

Reply:
Fixed in [9be3da9](https://github.com/medibloc/panacea-core/pull/486/commits/9be3da9de8b7d7787d0cecf1a4c8d022d5dd02b3).